sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClient/DataFeedLiveTests.cs

```diff
@@ -317,457 +313,13 @@ public async Task CreateAndGetSqlServerDataFeedWithCredentialAuthentication(stri
         }
 
         [RecordedTest]
-        public async Task UpdateAzureApplicationInsightsDataFeedWithMinimumSetup()
```

Comment:
We have a lot repeated code in this file, so we merged some of it into new tests.

- All `Update<data-feed-kind>WithMinimumSetup` tests (13) have been merged into a single `UpdateWithMinimumSetup` test. There's also a new separate test called `UpdateWithTokenCredential` for AAD testing.
- All `Update<data-feed-kind>WithEveryMember` tests (13) have been merged into a single `UpdateWithOptionalMembers`.